tutorial.md

```diff
@@ -0,0 +1,108 @@
+# Garden quickstart for Cloud Shell
+
+## Let's get started
+
+This quickstart will show you how to start a minikube cluster and deploy a simple application to it using Garden.
+
+* **Setup your environment**
+  * Install Garden and start a k3d Kubernetes cluster you'll use to deploy your app to.
+* **Deploy the app**
+  * You will be guided through the process of deploying a simple app to the cluster using Garden.
+* **Explore the dev console**
+  * Run tests, view logs, and more.
+___
+
+**Time to complete**: <walkthrough-tutorial-duration duration=10></walkthrough-tutorial-duration>
+Click the **Start** button to move to the next step.
+
+## Setup your environment
+
+### Open the Cloud Shell
+
+Open Cloud Shell by clicking
+<walkthrough-cloud-shell-icon></walkthrough-cloud-shell-icon> in the navigation bar at the top of the console.
+
+### Run the setup script
+
+Run the start script to prepare the environment. The script will install the latest version of `garden` and its dependencies, and start a local k3d Kubernetes cluster.
+
+```bash
+chmod +x start.sh && ./start.sh
+```
+
+## Deploy the app
+
+Garden ships with an interactive command center we call the **dev console**. To start the dev console, run:
+
+```sh
+garden dev
+```
+
+The first time you run `garden dev`, Garden will initialize then await further instructions inside a [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop). From inside the REPL you can command Garden to build, test, and deploy your project.
+
+After running `garden dev`, you're ready to deploy your project. Run:
+
+```sh
+deploy
+```
+
+If you receive an error like the following, wait until the `deploy` has finished, then try again:
```

Comment:
Why are we getting this error in the first place? 🤔

Reply:
@10ko I have a bug opened [here](https://github.com/garden-io/garden/issues/4460) both @ShankyJS and I are running into. I've asked devs to take a look 👀

Reply:
Rendering bug has been fixed!